datahub-frontend/conf/routes

```diff
@@ -45,6 +45,7 @@ GET           /authenticate                                                   re
 POST          /logIn                                                          react.controllers.AuthenticationController.logIn()
 GET           /callback/oidc                                                  @org.pac4j.play.CallbackController.callback()
 POST          /callback/oidc                                                  @org.pac4j.play.CallbackController.callback()
+GET           /centralLogout                                                  controllers.CentralLogoutController.executeLogout()
```

Comment:
What if we just call this "/logOut" for simplicity? The controller can figure out how to do that based on what auth mechanisms are enabled

Reply:
Yes, I will make the change shortly